tests/Rector/Contrib/FrameworkMigration/LaravelToSymfony/Wrong/wrong.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Response;
+
+class UserController extends Controller
+{
+    public function show(): Response
```

Comment:
This return type is wrong. It should be `Illuminate\View\View`

Reply:
Ok. I just copied example from the docs: https://laravel.com/docs/5.6/controllers#basic-controllers

Could you maybe send gist of what is common Laravel controller looks like? It'd be really helpful, since I don't use it at all

Reply:
Strange. The `view` helper method returns an `Illuminate\View\View`. The `json` a `Illuminate\Http\JsonResponse`. The `response` that only returns a `Response`